src/server/routes/entity/entity.js

```diff
@@ -364,6 +366,10 @@ async function processEditionSets(
 	);
 
 	const releaseEvents = _.get(body, 'releaseEvents') || [];
+	if (!releaseEvents[0].areaId) {
+		releaseEvents[0].areaId = null;
+	}
+
```

Comment:
https://github.com/bookbrainz/bookbrainz-data-js/blob/9faa67b5631ea5816828e05680e579d11526515f/src/func/releaseEvent.js#L34
it was returning false when old and new areaId were 'null' and ;undefined.;
I guess better solution would be if we change the bookbrainz-data code accordingly. But this works for now.

Reply:
The releaseEvents is always an array of size 1 right?

Reply:
At the moment yes.
If you want that bit of code to be more error-proof, you could do something like
`if (_.isNil(_.get(releaseEvents,'[0].areaId')) {…`

_.get returns the value at the path safely (won't throw an error if there's no `releaseEvents[0]` for example), and _.isNil determines if the value is null or undefined (instead of falsy; for example in this case if areaId = 0 it will be considered falsy and be overwritten with null [not sure that will ever happen, but…])

Reply:
I just created this as well as you suggested, if you want to review it:
https://github.com/bookbrainz/bookbrainz-data-js/pull/212

Reply:
Now that you've made changes in bookbrainz-data, this is not needed. 
Should I remove it or leave it untill those changes you made are merged in the master?

Reply:
Let's leave it, it doesn't hurt to be clean on both sides.